src/ontology/mondo-edit.obo

```diff
@@ -445257,10 +445258,10 @@ xref: SCTID:190505004 {source="DOID:12252"}
 xref: SCTID:47270006 {source="DOID:12252"}
 xref: UMLS:C0010481 {source="MEDGEN:3681", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0002254 {source="NCIT:C2969/inferred"} ! syndromic disease
-is_a: MONDO:0005495 {source="Orphanet:553"} ! adrenal gland disorder
```

Comment:
@MeeSiing we do not just remove SC, we also need to "exclude them" 
See documentation here: https://mondo.readthedocs.io/en/latest/editors-guide/g-logical-axioms/#excluded-subclassof
(the documentation also needs to be updated. Please also add your ORCID as a way to note that you have reviewed this SC exclusion)